cli/src/lib.rs

```diff
@@ -1,4 +1,7 @@
+// Macros need to be defined before used
+#[macro_use]
+pub mod logger;
+
```

Comment:
nit: do we need this empty line?

Reply:
Added just for visual separation of macros / mods. Can be removed